Makefile

```diff
@@ -125,8 +125,11 @@ BUILDROOT/ecs-agent.tar:
 
 .generic-rpm-done:
 	./scripts/update-version.sh
+	curl -O ${AGENT_URL}
```

Comment:
Hi @sparrc,

Thank you for your comment!  Unless someone else can fix the generic RPM build, I am going to close this PR and open a new one like my original one (https://github.com/aws/amazon-ecs-init/pull/432) with only the generic deb changes.  I see that there have been quite a few changes to the amazon-linux RPM build since opening this one, but I unfortunately do not have the time to read any deeper into the RPM build system.